src/main/java/com/checkmarx/flow/service/FlowService.java

```diff
@@ -37,10 +37,13 @@ private void runScanRequest(ScanRequest scanRequest, List<VulnerabilityScanner>
 
         scanners.forEach(scanner -> {
             ScanResults scanResults = scanner.scan(scanRequest);
-            combinedResults.mergeWith(scanResults);
+            if(scanResults!=null) {
```

Comment:
It would be cleaner to include the null check into mergeWith() method itself. mergeWith is in SDK though, so this change is up to you.

Reply:
Moved this check to sdk where it makes more sense.